Cargo.toml

```diff
@@ -542,6 +542,7 @@ default-members = [
 
 [workspace.lints.rust]
 suspicious_double_ref_op = { level = "allow", priority = 2 }
+unexpected_cfgs = { level = "allow", check-cfg = ['cfg(substrate_runtime)'] }
```

Comment:
What does this do?

Reply:
it allows crate to use the condition `substrate_runtime` in `cfg` without warning like this:

```
warning: unexpected `cfg` condition name: `substrate_runtime`
   --> substrate/primitives/runtime-interface/src/lib.rs:378:66
    |
378 | #[cfg(all(any(target_arch = "riscv32", target_arch = "riscv64"), substrate_runtime))]
    |                                                                  ^^^^^^^^^^^^^^^^^
    |
    = help: expected names are: `clippy`, `debug_assertions`, `doc`, `docsrs`, `doctest`, `feature`, `miri`, `overflow_checks`, `panic`, `proc_macro`, `relocation_model`, `rustfmt`, `sanitize`, `sanitizer_cfi_generalize_pointers`, `sanitizer_cfi_normalize_integers`, `target_abi`, `target_arch`, `target_endian`, `target_env`, `target_family`, `target_feature`, `target_has_atomic`, `target_has_atomic_equal_alignment`, `target_has_atomic_load_store`, `target_os`, `target_pointer_width`, `target_thread_local`, `target_vendor`, `test`, `ub_checks`, `unix`, and `windows`
    = help: consider using a Cargo feature instead
    = help: or consider adding in `Cargo.toml` the `check-cfg` lint config for the lint:
             [lints.rust]
             unexpected_cfgs = { level = "warn", check-cfg = ['cfg(substrate_runtime)'] }
    = help: or consider adding `println!("cargo::rustc-check-cfg=cfg(substrate_runtime)");` to the top of the `build.rs`
    = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration
    = note: `#[warn(unexpected_cfgs)]` on by default

```

Reply:
CLippy...